pyproject.toml

```diff
@@ -22,11 +22,7 @@ commands =
 changedir = examples
 allowlist_externals = /bin/bash
 
-deps =
-    jupyter
-    jupyter_contrib_nbextensions
-    matplotlib
-    tqdm
+deps = -rexamples/requirements.txt
```

Comment:
Sorry, I was not very clear. I meant moving only the dependencies required to run the notebook in the requirements.txt (i.e. matplotlib & tqdm), keeping the requirements for the tests here (jupyter and jupyter_contrib_nbextensions)

Reply:
but jupyter and jupyter_contrib_nbextensions is only required to run the notebooks in the examples, it is not required for the tests

Reply:
So the "convert to python" we are using below is coming from jupyter itself? That's neat. Then I guess this is good to go!

Reply:
if you mean nbconvert, yes!